spec/system/jobseekers_can_complete_a_job_application_spec.rb

```diff
@@ -4,15 +4,17 @@
   let(:jobseeker) { create(:jobseeker) }
   let(:vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: organisation }]) }
   let(:organisation) { create(:school) }
-  let(:job_application) { create(:job_application, jobseeker: jobseeker, vacancy: vacancy) }
 
   before do
     allow(JobseekerApplicationsFeature).to receive(:enabled?).and_return(true)
     login_as(jobseeker, scope: :jobseeker)
   end
 
-  it "allows jobseekers to complete application and go to review page" do
-    visit jobseekers_job_application_build_path(job_application, :personal_details)
+  it "allows jobseekers to complete an application and go to review page" do
+    visit job_path(vacancy)
```

Comment:
We have `jobseekers_can_start_a_job_application_spec.rb`, so I'd prefer to leave this as it is unless you can compel me otherwise!